Mathlib/LinearAlgebra/Dimension/Basic.lean

```diff
@@ -333,7 +334,7 @@ theorem rank_range_of_surjective (f : M →ₗ[R] M') (h : Surjective f) :
 
 theorem rank_submodule_le (s : Submodule R M) : Module.rank R s ≤ Module.rank R M := by
   rw [← rank_top R M]
-  exact rank_le_of_submodule _ _ le_top
```

Comment:
Should this lemma be called `Submodule.rank_le` instead? You're renaming the lemma above to `Submodule.rank_mono`...

Actually: the analogous lemma for finrank is called `Submodule.finrank_le` - please rename.

Reply:
Renamed